doc/cpd/101.rst

```diff
@@ -75,6 +78,16 @@ if set to ``True`` would use the secrets handler.
 The secrets handler would be automatically added to the ``StorageHandlerManager`` and
 would require no special configuration by the operator.
 
+New HTTP Handler
+~~~~~~~~~~~~~~~~
+A new handler called ``secrets`` could be added. This would proxy requests back to the
+Custodia instance.
```

Comment:
Correct. Those would be used as the auth into custodia through our endpoint. Since we'd just be proxying access and allowing custodia to handle it's own access we'd need to add the auth changes listed.